specification/metadata/focusversion.md

```diff
@@ -0,0 +1,17 @@
+# FOCUS Version
```

Comment:
We also need a DataGeneratorVersion (or something like that) to account for data generators creating multiple versions of their support of a specific FOCUS version (e.g., with new custom columns).

Reply:
Added DataGeneratorVersion